src/workerd/api/http-test.js

```diff
@@ -206,44 +206,56 @@ export const inspect = {
   }
 };
 
+async function requestCacheThrowsError(cacheHeader, errorName, errorMessage){
+  assert.throws(() => {
+      new Request('https://example.org', { cache: cacheHeader });
+    }, {
+      name: errorName,
+      message: errorMessage,
+    }
+  );
+}
+
+async function requestCacheThrowsDefaultError(cacheHeader) {
+  requestCacheThrowsError(
+    cacheHeader,
+    'Error',
```

Comment:
could these be default argument values instead?

Reply:
Yes.